src/pluginManager/browser.ts

```diff
@@ -19,8 +19,13 @@ import { consoleReportingPlugin, reportingPluginsWrapper } from '../plugins/repo
 import defaultTransitionHooksPlugin from '../plugins/transitionHooks/browser';
 
 export default class PluginManager extends CommonPluginManager implements PluginManagerPublicApi {
-    constructor(...contexts: Array<Context>) {
-        super(['transitionHooks', 'reporting'], ...contexts);
+    /**
+     * @deprecated use: new PluginManager(...plugins: Plugins[]) instead
+     */
+    constructor(...plugins: Context[]);
+    constructor(...plugins: Plugin[]);
+    constructor(...plugins: Plugin[] | Context[]) {
```

Comment:
Plugin[] | Context[] Looks like it duplicates 
```
constructor(...plugins: Context[]);
    constructor(...plugins: Plugin[]);
```

Reply:
No it is not. Required by TS: as far as method overload doesn't allow to have miltiple implementations all possible values should be included for single implementation. See [playground](https://www.typescriptlang.org/play?#code/FAYwNghgzlAEBiB7RAjCAnWBvWxb9gHoAqYvA2Y2AAQBMBTAB3XpAgBd7bYBXKe2ADt6AdwTI06ABQQAXEJ4BbFPXQBKWAEtBUThFrkCxQodghEO9uh4h2iaXNi702gOZqA3KfOXrt+zLygkoq6l4UPs42dg7yzm6wAD4KyqoaWMCmAL7AOaCQMLAAQhhIiNimRKSVVHRMLGyc3HwCwmJlkoEpoRrauvT6NSYRFlH+sU5Wbp7eo1bRAY7BqeoVFBTZuZmgc7AAZsglmAC8QqLiqBhSAOQArNczQA).